apps/meteor/client/components/CreateDiscussion/CreateDiscussion.tsx

```
@@ -132,6 +135,21 @@ const CreateDiscussion = ({
+									value.trim().length > 0 || t('Required_field', { field: t('Name') }),
+								}}
+								render={({ field }) => (
+									<TextInput
+										id={discussionNameId}
+										{...field}
+										aria-invalid={Boolean(errors.name)}
+										aria-required='true'
+										aria-describedby={`${discussionNameId}-error ${discussionNameId}-hint`}
+										addon={<Icon name='baloons' size='x20' />}
+									/>
+								)}
```

Biome (2.3.13)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)